stx-genesis/src/lib.rs

```diff
@@ -175,4 +218,14 @@ mod tests {
             assert!(name.owner.len() > 0);
         }
     }
+
+    #[test]
+    fn test_name_zonefiles_read() {
+        for zonefile in GenesisData::new(false).read_name_zonefiles() {
+            assert_eq!(zonefile.zonefile_hash.len(), 40);
```

Comment:
Do you want to check at least that the zone file contents are correct?

Reply:
I think the build itself fails if the zone file contents don't correspond to the sha sum.

Reply:
Yeah these tests are just checks against the sampled genesis.